bin/addons-linter

```diff
@@ -5,6 +5,13 @@ var path = require('path');
 var absoluteAppRoot = path.resolve(path.join(__dirname, '..'));
 global.appRoot = path.relative(process.cwd(), absoluteAppRoot);
 
-require('../dist/addons-linter')
-  .createInstance({runAsBinary: true})
-  .run();
+try {
+    require('../dist/addons-linter')
+       .createInstance({runAsBinary: true})
+        .run();
+} catch (e) {
+    if (e.code == 'MODULE_NOT_FOUND') {
+        console.log("Please refer https://github.com/mozilla/addons-linter#readme");
```

Comment:
I would add a bit more information to it. Let's do maybe something like `You did not build addons-linter yet. Please run 'npm install' and 'yarn build' or 'npm build' or see https://github.com/mozilla/addons-linter#readme for more information` cc @tofumatt for wording.

Let's also use single-quotes as everywhere else in the module.